### ISSUE TITLE
[idea] apk package manager

### ISSUE THREAD

**Issue by awesomelewis2007** [ish-app/ish#1726]
It would be quite cool to have a package manager for ish like in Debian 

![image](https://user-images.githubusercontent.com/60709927/154940979-cc73533b-2b2e-4c41-8b7f-dc55f1af5902.png)

I liked the update repository's button it would be cool if there was a install packages button

**Comment by emkey1** [comment]
There is a text based update manager on in Settings on newer versions of iSH.  I'm not sure if this is true of the current App Store version, but is in the TestFlight version for sure.

**Comment by saagarjha** [comment]
This is shipping in the latest version on the App Store.

**Comment by awesomelewis2007** [comment]
> There is a text based update manager on in Settings on newer versions of iSH.  I'm not sure if this is true of the current App Store version, but is in the TestFlight version for sure.

Yeah but I'm talking about a package manager not an updater

**Comment by Rhinozz** [comment]
I don't believe Alpine holds icons nor descriptions of its packages - this seems like unnecessary work for something that is effectively `apk search`. Additionally, the Linux community's very negative opinion on QApt makes me think that this isn't a very high-demand feature either.

**Comment by awesomelewis2007** [comment]
> I don't believe Alpine holds icons nor descriptions of its packages - this seems like unnecessary work for something that is effectively `apk search`. Additionally, the Linux community's very negative opinion on QApt makes me think that this isn't a very high-demand feature either.

No I'm not expecting icons and fancy graphics I just want a simple package manager it could be a checklist of packages that you check and at the top it would say install
